custom_model_runner/requirements.txt

```diff
@@ -13,4 +13,4 @@ scipy>=1.1,<2
 strictyaml==1.0.6
 texttable
 py4j~=0.10.9.0
-pyarrow
\ No newline at end of file
+pyarrow
```

Comment:
@yakov-g @4dahalibut I've made `pyarrow` required again (don't know why this diff is even shown)
probably we should not pin `pyarrow` version - we don't care about any particular version, and there're more chances it'll get installed if unpinned

so, is this PR good to go?

Reply:
1. I'd go without `pyarrow`, but @4dahalibut and @jlpolit should tell how many people use transform.
2. If we go with `pyarrow`, we should not pin it, as new pip with deps check would scream, if requirements are not met.
3. I still need to add a test case for R+mtx. So @4dahalibut or @jlpolit please approve, and i'll merge once test is in.

Reply:
Lots of people will be using transforms. I'm waiting on at least sparse R testing to continue.

Reply:
So...
mtx case is implicitly tested in `test_fit_sparse`, when model is verified after fitting.

To add such test case into inference test, I'll need to add new R artifact, trained on sparse data. I can not use R artifact trained on csv data with sparse payload as it is missing column names.

Do you want me to add an artifact and separate test casr for R with mtx?

Reply:
@4dahalibut ^

Reply:
@4dahalibut added the test case for R + mtx

Reply:
I still think that this needs to be pinned at least in some range. having unpinned deps is really not good.

Reply:
The thing is - I don't know which policy to apply for pinning it.
These are our requirements:
```
argcomplete==1.11.1
datarobot==2.21.3
docker>=4.2.2<5.0.0
flask
jinja2
memory_profiler<1.0.0
mlpiper~=2.4.0
numpy
pandas
progress
requests
scipy>=1.1,<2
strictyaml==1.0.6
texttable
py4j~=0.10.9.0
pyarrow
```
We pin only packages essential for drum. ML packages can change from user to user and we don't pin them, assuming (probably that's bad), that we use just basic functionality, like reading DF.

Maybe we should go over everything and pin versions to  `> than`

In particular, for arrow, we use 0.14.1 for DR, while the latest is 2.0.0. 
They are more than a year apart, so pinning to `pyarrow>0.14.1` is like not pinning at all.